Makefile

```diff
@@ -938,6 +938,19 @@ geojson/districts.geojson: shp/us/congress-ungrouped.shp
 	cat $(dir $@)temp.json | ./clip-at-dateline > $@
 	rm $(dir $@)temp.json
 
+geojson/us-10m/%.geojson: geojson/%.geojson
+	node_modules/.bin/topojson \
+		-o temp.json \
+		--no-pre-quantization \
+		--post-quantization=1e6 \
+		--simplify=7e-6 \
+		--properties GEOID,STATE_FIPS,FIPS\
+		--external-properties fips.csv \
+		-- $^
+	node_modules/.bin/topojson-geojson -o $(dir $@) temp.json
+	mv $(basename $@).json $@
+	rm temp.json
```

Comment:
@dereklieu ^ this creates a topojson from the full-res boundaries, simplifying the shapes while preserving topology (i.e. shared edges)